<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddForm</class>
 <widget class="QWidget" name="AddForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>531</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="patient">
    <attribute name="title">
     <string>Пациенты</string>
    </attribute>
    <widget class="QTextEdit" name="familyTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="nameTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="fatherTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="fatherLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Отчество</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QLabel" name="familyLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
    <widget class="QLabel" name="IDlabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID пациента</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="IDspinBox">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>11</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QGroupBox" name="sexBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Пол</string>
     </property>
     <widget class="QRadioButton" name="maleButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Мужской</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="femaleButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Женский</string>
      </property>
     </widget>
    </widget>
    <widget class="QDateEdit" name="birthDateEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="birthDateLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Дата рождения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="phoneTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>249</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="phoneLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Телефон</string>
     </property>
    </widget>
    <widget class="QLabel" name="registrationDateLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Дата регистрации</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="registrationDateEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addPatientButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Добавить запись</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="addressLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Адрес проживания</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="addressTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>209</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="appointment">
    <attribute name="title">
     <string>Приёмы</string>
    </attribute>
    <widget class="QLabel" name="cabinetLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Кабинет</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="reasonTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="doctorLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Лечащий врач</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="doctorTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="cabinetTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="appointmentDateLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Дата обращения</string>
     </property>
    </widget>
    <widget class="QLabel" name="appointmentIDlabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID пациента</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="appointmentDateEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>91</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="reasonLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Причина обращения</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="appointmentTimeEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="appointmentTimeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Время обращения</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addAppointmentButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Добавить запись</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="appointmentIDTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="appIDTextEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="appIDlabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID приёма</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="doctor">
    <attribute name="title">
     <string>Врачи</string>
    </attribute>
    <widget class="QLabel" name="doctorFamilyLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="doctorNameEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="doctorFatherLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Отчество</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="doctorFatherEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="doctorFamilyEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="specialLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
    <widget class="QLabel" name="doctorNameLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="specialEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>130</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addDoctorButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Добавить запись</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
